book/website/communication/peer-review/peer-review-guidance.md

```diff
@@ -81,30 +82,43 @@ Also be mindful of disciplinary and cultural differences between the authors and
 If you searched for literature, checked the data and verified the results, ran additional analyses, state this. 
 It will allow the editor to adjudicate the review.
 2. Then give a brief overview of the paper. 
-If the invitation asks you to provide a general recommendation, state it early in the review.
+Include a two- or three-sentence summary with your interpretation of the hypothesis, main results and conclusions. 
+This summary provides the author an indication of the clarity of their paper. 
+3. If the invitation asks you to provide a general recommendation, state it early in the review.
 The recommendation is the first thing the editor wants to know.
 Typically, you are invited to recommend ‘reject’, ‘revise & resubmit’ – with major or minor revisions, or ‘accept’. 
-3. Use ‘reject’ only if absolutely necessary.
+  - Major revisions are needed when the conclusions are jeopardized by concerns or the usefullness of the data is limited. 
+  Examples are problems with the applied methods, analyses, or the reasoning. 
+  Reviewers should indicate the seriousness of the major concern and how it could be addressed. 
+  If the concern cannot be addressed it may be better to 'reject' the work. 
+  Only suggest additional analyses or testing only when the conclusions of the study are otherwise undermined - otherwise they can be addressed in a future paper. 
+  - Minor revisions are needed to improve the clarity of the paper. 
+  Minor revisions can include additional citations, addition of details regarding the method, change in title or abstract to reflect the paper content, whether figures are clear, necessairy and in correspondence with the text and any presentation issues (spelling errors, wording). 
+4. Use ‘reject’ only if absolutely necessary.
 Most journal editors make use of desk-rejection if the paper seems a poor fit; if the editor sent the paper out for review, she probably thinks it has the potential to be published.
 Start from the assumption that the authors have invested a great deal of time in the paper and that they want to improve it. 
-4. List the strengths and the weaknesses of the paper. 
+5. List the strengths and the weaknesses of the paper. 
 To ease the message you can use the sandwich principle: start with the strengths, then discuss the weaknesses, and conclude with an encouragement.
-5. Try to give actionable advice. 
+6. Try to give actionable advice. 
 For the weaknesses in the paper, suggest ways to repair them. 
```

Comment:
Constructive criticism does not have to be in the form of a specific solution. It can be much more useful to be specific about the problem and to let the authors identify a solution, than it is to provide the fix for them.

Reply:
Added, thanks!